website/layouts/partials/head.html

```diff
@@ -0,0 +1,2 @@
+<link rel="me" href="https://x.com/@TheLimaProject" />
+<link rel="me" href="https://mastodon.social/@TheLimaProject" />
```

Comment:
This seems to break the HTML 🤷‍♂️ 
https://deploy-preview-4445--lima-vm.netlify.app